runtime/compiler/il/J9Node.hpp

```diff
@@ -441,8 +443,8 @@ class OMR_EXTENSIBLE Node : public OMR::NodeConnector
 private:
 
 #ifdef TR_TARGET_S390
-   ///< hasStorageReferenceHint()
-   TR_StorageReference * _storageReferenceHint;
+   //< hasStorageReferenceHint()
+      TR_StorageReference * _storageReferenceHint;     ///< hasStorageReferenceHint()
```

Comment:
Please remove the three spaces that were added before `TR_StorageReference`.  Also, please remove the trailing comment that was added on this line